### PR TITLE
Talos2 2nd cpu

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+to Skiboot to include information about new CPU and devices it adds. We also
+switched away from FIT payload and started using ELF. Main motivation for this
+change was fact that [FIT decompression code does not work as expected](https://mail.coreboot.org/hyperkitty/list/coreboot@coreboot.org/thread/6EZWU7YPUJE564GNCV7U32IXWPFTV7FB/)
+in coreboot. With payload compressed with LZMA, flash memory footprint of whole
+coreboot and Skiboot has shrunk to just below 700 KB. For comparison, Hostboot
+takes about 30 MB of space.
+
+## New tests
+
+TBD
```
Comment (macpijan) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:83]: @sulewskiprzemyslaw We would expect short description of changes in the test area here.

**Review by SergiiDmytruk — APPROVED** [review]

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+switched away from FIT payload and started using ELF. Main motivation for this
+change was fact that [FIT decompression code does not work as expected](https://mail.coreboot.org/hyperkitty/list/coreboot@coreboot.org/thread/6EZWU7YPUJE564GNCV7U32IXWPFTV7FB/)
+in coreboot. With payload compressed with LZMA, flash memory footprint of whole
+coreboot and Skiboot has shrunk to just below 700 KB. For comparison, Hostboot
+takes about 30 MB of space.
+
+## New tests
+
+In order to achive greater reliability of the firmware and to avoid bugs test
+base on the TalosII platform has been improved.
```
Comment (krystian-hebel) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:84]:
```suggestion
base on the Talos II platform has been improved.
```

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+[HDS]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31U-heads-bootloader-support/
+[USB]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31O-usb-detect/
+[UBB]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31N-usb-boot/
+[LBT]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/308-debian-stable-and-ubuntu-lts-support/
+[DVT]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31W-device-tree/
+[CPU]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31T-cpu-status/
+[Tests]: https://docs.dasharo.com/unified-test-documentation/overview/
+[Matrix]: https://docs.dasharo.com/variants/talos_2/test-matrix/
+
+Description of all dasharo test cases
```
Comment (krystian-hebel) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:122]: ???

Comment (sulewskiprzemyslaw) [reply blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md]: Redundant line. I just deleted it

**Review by sulewskiprzemyslaw — APPROVED** [review]

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Currently, before each firmware release, the following test suites will be
+performed:
+- [Dasharo compatibility: coreboot base port][CBP], which contains test cases
+  for checking correctness of coreboot porting on the device.
+- [Dasharo compatibility: Petitboot payload support][PBT], which contains test
+  cases for checking Petitboot availabity and functionality.
+- [Dasharo compatibility: Heads bootloader support][HDS], which contains test
+  cases for checking Heads bootloader availabity.
+- [Dasharo compatibility: USB detection][USB], which contains test cases for
+  checking correctness of USB detection after coldboot, warmoot and system
```
Comment (SergiiDmytruk) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:95]:
```suggestion
  checking correctness of USB detection after coldboot, warmboot and system
```

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  reboot.
+- [Dasharo compatibility: USB booting][UBB], which contains tests cases for
+  checking correctness of booting from USB after coldboot, warmboot and system
+  reboot.
+- [Dasharo compatibility: Debian Stable and Ubuntu LTS support][LBT], which
+  contains test cases for checking correctness of installing and booting into
+  installed OS.
+- [Dasharo compatibility: Device Tree][DVT], which contains test cases for
+  checking correctness of presentation coreboot in Device Tree.
```
Comment (SergiiDmytruk) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:104]:
```suggestion
  checking correctness of coreboot presentation in Device Tree.
```

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+[UBB]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31N-usb-boot/
+[LBT]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/308-debian-stable-and-ubuntu-lts-support/
+[DVT]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31W-device-tree/
+[CPU]: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31T-cpu-status/
+[Tests]: https://docs.dasharo.com/unified-test-documentation/overview/
+[Matrix]: https://docs.dasharo.com/variants/talos_2/test-matrix/
+
+## Summary
+
+TBD
```
Comment (macpijan) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:124]: @krystian-hebel What do we want to include here?

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+---
+
+Another post about our adventures with porting coreboot for Talos II. This phase
+focused on enabling second CPU and its internal devices. We also expanded our
+test suite.
+
+## Links to binaries and installation instructions
+
+TBD
```
Comment (macpijan) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:28]: @krystian-hebel We already know the links. Maybe we can just link to the relevent section in the documentation?

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md (hunk 6 above)
Comment (krystian-hebel) [reply blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md]: Added

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md (hunk 7 above)
Comment (krystian-hebel) [reply blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md]: Added

File: blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
+---
+title: Talos II - second CPU support and test automation
+abstract: 'Another post about our adventures with porting coreboot for Talos II.
+          This phase focused on enabling second CPU and its internal devices.
+          We also expanded our test suite.'
+cover: /covers/coreboot-logo.svg
+author: krystian.hebel
+layout: post
+published: true
+date: 2022-04-12
```
Comment (macpijan) [inline blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md:10]:
```suggestion
date: 2022-04-15
```

**Pull request merged**